class-debug-bar-cron.php

```diff
@@ -380,6 +381,17 @@ private function display_schedules() {
 				</table>';
 		}
 
+		/**
+		 * Sorting method for cron scheldules. Order by scheldule interval
+		 *
+		 * @param array $a first element of compasion  pair
+		 * @param array $b second element of compasion  pair
+		 *
+		 * @return int 1 if element $a interval greater then element $b interval, -1  otherwise.
+		 */
+		function schedules_sorting($a, $b){
+			return (int) $a['interval'] > $b['interval'];
```

Comment:
Just thinking out loud: what about adding a `ksort()` just before the `uasort()` so that - no matter the order in which they were added -, the secondary sort wil be on the hook name ?

This would require the callback to return `0` when the values are equal I believe.